doc/source/CreatingEnvironments.rst

```diff
@@ -0,0 +1,33 @@
+.. _CreatingEnvironment:
+
+Creating and installing new environments
+****************************************
+
+The following instructions are a simplified version of the instructions found in :numref:`Sections %s <NewSiteConfigs_macOS>` and :numref:`%s <NewSiteConfigs_Linux>`, and should work for systems where spack-stack already has a site configuration, or where minimal configuration is necessary. For more detailed instructions, including how to generate new site configurations and more details of when and how to use external packages, see :numref:`Sections %s <NewSiteConfigs_macOS>` and :numref:`%s <NewSiteConfigs_Linux>`. To chain a new Spack environment to an existing one, such as to test a new package version based on dependencies already installed in the upstream environment, see :numref:`%s <Add_Test_Packages>`.
+
+Note that items in "<>" should be replaced with the appropriate values (site names, etc.), and items in '[]' are optional. *<site name>* and *<template name>* must match the names of directories found under 'configs/sites/' and 'configs/templates/', respectively; *<environment name>* is a user-chosen name for the Spack environment. *<path to spack-stack directory>* is the root directory created by cloning the spack-stack GitHub repository, which contains 'configs/', 'doc/', 'setup.sh', 'util/', etc.
+
+.. code-block:: console
+
+   # To generate a new spack-stack directory structure, run 'git clone --recurse-submodules https://github.com/JCSDA/spack-stack',
+   # optionally with, e.g., '-b release/1.4.1' to specify the version
+
+   . <path to spack-stack directory>/setup.sh
+
+   spack stack create env --site <site name> --template <template name> --name <environment name>
+
+   cd ${SPACK_STACK_DIR}/envs/<environment name>/
```

Comment:
What would you think of moving to putting log.{concretize,install} under the environment itself? That way the root dir doesn't get cluttered, especially if we install additional environments.

Reply:
Hmm, it's super convenient to never have to `cd` elsewhere except the spack-stack top-level directory. I also found it useful to see what kind of envs exist and what has been installed. Anyway, no strong preference, but it will require some training since I've more or less hardcoded those steps in my finger muscles (and in my scripts)!

Reply:
I have a moderate preference toward putting the logs in their corresponding env directories for organizational reasons, but no strong preference on the cd. Would you prefer, say, `... | tee envs/unified-env/log.concretize`?

Reply:
I'll leave it up to you - whatever is easier for everyone and less typing.